roles/prereq/tasks/main.yml

```diff
@@ -1,8 +1,13 @@
 ---
+- name: Set same timezone on every Server
+  timezone:
+    name: "{{ system_timezone }}"
+  when: (system_timezone is defined) and (system_timezone != "Your/Timezone")
```

Comment:
I see, to this is optional.  If it's the default or not defined it will skip it?

Reply:
Hello Tim
Sorry I am not active on github that often.
Yes this when statement checks if the variable is defined and also if its not the default variable "Your/Timezone" (A dummy variable to escape the need for documentation). Else it is gonna skip